dejagnu/ChangeLog
	* lib/target.exp (prune_warnings): Correct pic/PIC regexp.
	Ignore incompatible library warning.

Index: dejagnu/lib/target.exp
===================================================================
RCS file: /cvs/src/src/dejagnu/lib/target.exp,v
retrieving revision 1.12
diff -u -p -r1.12 target.exp
--- dejagnu/lib/target.exp	21 Apr 2002 08:47:07 -0000	1.12
+++ dejagnu/lib/target.exp	9 Jun 2003 07:34:06 -0000
@@ -279,7 +279,10 @@ proc prune_warnings { text } {
     regsub -all "(^|\n)\[^\n\]*: warnidejagnu/ChangeLog
	* lib/target.exp (prune_warnings): Correct pic/PIC regexp.
	Ignore incompatible library warning.

Index: dejagnu/lib/target.exp
===================================================================
RCS file: /cvs/src/src/dejagnu/lib/target.exp,v
retrieving revision 1.12
diff -u -p -r1.12 target.exp
--- dejagnu/lib/target.exp	21 Apr 2002 08:47:07 -0000	1.12
+++ dejagnu/lib/target.exp	9 Jun 2003 07:34:06 -0000
@@ -279,7 +279,10 @@ proc prune_warnings { text } {
     regsub -all "(^|\n)\[^\n\]*: warning:   as it has already been specified\[^\n\]*" $text "" text
 
     # Cygwin cc1 warns a